tests/acceptance/README

```diff
@@ -79,12 +79,20 @@ or directories to 'testall':
 Creating/editing test cases
 ------------------------------------------------------------------------------
 
-Each test should be 100% standalone, and must contain at least 1 of the main
-bundles:
+Each test should be 100% standalone. If you include "default.cf.sub" in
+inputs, then the bundlesequence is automatically defined, and your file
+must contain at least 1 of the main bundles:
 
 * init - setup, create initial and hoped-for final states
 * test - the actual test code
 * check - the comparison of expected and actual results
+* destroy - anything you want to do at the end like killing processes etc
+
+However if the test is really simple you might want to skip including
+"default.cf.sub" in inputs. Then you should define your own
+bundlesequence, e.g. {"test","check"}. It's recommended to avoid
+including "default.cf.sub" when not needed, since the test will run much
+faster.
```

Comment:
Yes it is much faster, our `default.cf.sub` has grown to be quite the beast. Ironically the reason we don't use our library bodies from `masterfiles/lib` is because it was deemed too big and it was slowing down the tests.

Can I haz compiled policy files?